tests/test_download.py

```diff
@@ -1,23 +1,67 @@
+import hashlib
+from io import BytesIO
+
 import pytest
+import requests
+from PIL import Image
 
 from streetview import get_panorama
-from streetview.download import iter_tile_info, iter_tiles
+from streetview.download import (
+    get_width_and_height_from_zoom,
+    iter_tile_info,
+    iter_tiles,
+    make_download_url,
+)
+
+# This MD5 was retrieved empirically by downloading tile with bad coordinates
+BAD_TILE_MD5 = "be32aa9ed3880664433199f9e0615cd6"
+
+
+def get_tile_md5_from_url(pano_id: str, zoom: int, x: int, y: int):
+    """
+    Returns the md5 hash of a tile from url
+    """
+    url = make_download_url(pano_id=pano_id, zoom=zoom, x=x, y=y)
+    response = requests.get(url, stream=True)
+    image = Image.open(BytesIO(response.content))
+
+    image_file = BytesIO()
+    image.save(image_file, "jpeg")
+    image_file.seek(0)
+    return hashlib.md5(image_file.read()).hexdigest()
+
+
+@pytest.mark.parametrize("zoom", [1, 2, 3, 4, 5, 6, 7])
+def test_width_and_height_from_zoom_is_correct(zoom: int):
```

Comment:
This test makes HTTP calls, but I do not see a `@pytest.mark.vcr` on it.

Reply:
Done